nautobot/extras/views.py

```diff
@@ -800,9 +800,7 @@ def extra_context(self):
                 task_name__startswith="nautobot.core.jobs.GitRepository",
                 task_kwargs__repository__isnull=False,
                 status__in=JobResultStatusChoices.READY_STATES,
-            )
-            .order_by("date_done")
-            .defer("data")
+            ).order_by("date_done")
```

Comment:
Same concern here and below about performance implications of deleting the `defer()` calls.